code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -909,16 +915,33 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 			update_icons()
 		return
 
-/mob/living/silicon/robot/verb/unlock_own_cover()
+/mob/living/silicon/robot/verb/toggle_own_cover()
 	set category = "Robot Commands"
-	set name = "Unlock Cover"
-	set desc = "Unlocks your own cover if it is locked. You can not lock it again. A human will have to lock it for you."
-	if(locked)
-		switch(alert("You can not lock your cover again, are you sure?\n      (You can still ask for a human to lock it)", "Unlock Own Cover", "Yes", "No"))
-			if("Yes")
-				locked = 0
-				update_icons()
-				to_chat(usr, "You unlock your cover.")
+	set name = "Toggle Cover"
+	set desc = "Toggles the lock on your cover."
+	var/is_ert = has_special_laws()
+	//Standard borgs
+	if(!is_ert)
+		if(locked)
```

Comment:
This provides no feedback to normal borgs who click this while locked.

Reply:
Didn't think of adding a message since it didn't give one to begin with, but it's definitely better with one.